src/objects/charts/chart.data.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+const chartLabels = ['1980', '1990', '2000', '2010', '2020'];
+
+const chartDataList = [6542564, 7322564, 8008278, 8175133, 5475133];
+
+const ChartData = {
+  data: {
+    labels: chartLabels,
+    datasets: [
+      {
+        data: chartDataList,
+        fill: false, // used by line chart
+        backgroundColor: [
+          '#2F334F',
+          '#3155A6',
+          '#F2695D',
+          '#EBBCD8',
+          '#F9A137',
+        ],
+        borderColor: [
+          '#ACAEB9'
+        ],
+        borderWidth: 1
+      },
+    ]
+  },
+  bar: {
```

Comment:
It may make sense for default datasets attribute to live in this file but the configuration for the different chart types, display options, labels, etc. should probably live in either the chart type .js file or be divided into separate buckets.

Configuration buckets to consider creating;
- Chart Data: The raw data a user will want to feed into the chart.
- Chart Labels and Strings: These are user specified configurations.
- Chart Style: These are more or less hard coded attributes that determine the look and feel of the chart under the NYCO Brand. These are things like line thickness, padding, colors, etc. It looks like there may be different styles for different chart types!

It is possible to create a set of defaults for each of these buckets and use deep object assignment to override the them with user settings. I can share an example with the recent WordPress Vue Archive script for ACCESS that allows a user to pass in any Vue options to override the default Vue setting defined in the script.

Reply:
I'm now looking further into the different charts and it looks like this has already been considered!

Reply:
Yeah, unfortunately based on the way `vue-chartjs` handles reactivity, we can't set defaults for the dataset object. We can set defaults for the options object but I left that minimal for now. We can also override the defaults for the different chart types but I'm not sure if that's something we need to do (maybe setting the default color?). Happy to chat about more in person.